worker/src/handlers/post.ts

```diff
@@ -97,7 +97,7 @@ async function handlePost(request: Request, sentry: Toucan): Promise<Response> {
 
 async function storePayload(
   storageKey: string,
-  payload: IncomingPayload,
+  payload: any,
```

Comment:
this is still `IncomingPayload` right? Superstruct also tells typescript that an `unknown` is the correct type after a create or assert right?